File: COSC 4351/OO Assignment/DVDApplicationCSharp/DVD.cs
using System;



class DVD
{
    public string Title { get; set; }
    public string Category { get; set; }
    public int RunningTime { get; set; }
    public int Year { get; set; }
    public decimal Price { get; set; }

    public DVD(string title, string category, int runningTime, int year, decimal price)
    {
        Title = title;
        Category = category;
        RunningTime = runningTime;
        Year = year;
        Price = price;
    }
}

File: COSC 4351/OO Assignment/DVDApplicationPHP/DVD.php
<?php

class DVD
{
    public $Title;
    public $Category;
    public $RunningTime;
    public $Year;
    public $Price;

    public function __construct($title, $category, $runningTime, $year, $price)
    {
        $this->Title = $title;
        $this->Category = $category;
        $this->RunningTime = $runningTime;
        $this->Year = $year;
        $this->Price = $price;
    }
}